### DISCUSSION TITLE
Any way to track the mouse over a line chart regardless of mouse Y position?

### DISCUSSION THREAD

**Discussion by eamodio** [naver/billboard.js#3088] [Q&A]
I'm trying to get this chart below to track each date on the graph regardless of the Y position of the mouse. Is there any way to accomplish this?

![track-mouse](https://user-images.githubusercontent.com/641685/217164507-0977407e-b3a4-4794-8a06-9aaa936b58c5.gif)

**Comment by eamodio** [comment]
@netil sorry to ping you, but any idea on this?

**Comment by netil** [comment]
I assume that `tooltip.grouped=false` option is set. Try by removing or set as true.
`tooltip.grouped=false` option will make tooltip to appear on exact data's position.

**Reply by eamodio** [reply]
@netil Thanks for the reply!

Unfortunately, I tried setting `tooltip.grouped=true` and `selection.grouped=true` or any combination (either `true` or `false` for them).

And unless I set `point.sensitivity` to something large (have it at `100` currently) it won't even track well at all.  It seems like the vertical distance is causing the tracking to not work.

Here is my code if it helps: https://github.com/gitkraken/vscode-gitlens/blob/09c466d1885dc4cb27907820b1fd80e66d808c7e/src/webviews/apps/plus/graph/minimap/minimap.ts#L644-L893

**Comment by netil** [comment]
@eamodio, in normal use cases the event will be tracked on area which appears shape(line) regardless of y coordinate position.
Probably some combination of the option can make this happen.

I'll try reproduce the issue from the generation option you shared.

**Reply by eamodio** [reply]
Thank you!

**Comment by eamodio** [comment]
@netil I found what causes this issue -- its setting `xSort: false` in the options (which I unfortunately need).

**Reply by eamodio** [reply]
I opened: https://github.com/naver/billboard.js/issues/3100
